src/native/minipal/thread.h

```diff
@@ -70,7 +70,7 @@ static inline size_t minipal_get_current_thread_id(void)
 #elif defined(__HAIKU__)
         tid = (size_t)find_thread(NULL);
 #else
-        tid = (size_t)(void*)pthread_self();
+        tid = (size_t)pthread_self();
```

Comment:
browser CI legs are failing on this change. If it is needed for illumos you can wrap it in `#elif TARGET_SUNOS` and leave the `#else` as was.

Reply:
OK, thanks. I fixed that the pushed again.  It builds for SunOS.